src/reducers/versions.tsx

```diff
@@ -1253,8 +1259,23 @@ export const createReducer = ({
         };
       }
       case getType(actions.loadVersionInfo): {
-        const { version } = action.payload;
+        const { updatePathInfo, version } = action.payload;
         const internalVersion = createInternalVersion(version);
+
+        if (updatePathInfo === false) {
```

Comment:
I intentionally did this `updatePathInfo===false`, so that undefined `updatePathInfo` could be treated as `true`. It seems to be a bad idea, so I made some changes to set the default value explicitly.